src/clojure/nrepl/misc.clj

```diff
@@ -89,3 +90,28 @@
   []
   (.startsWith (System/getProperty "java.runtime.version")
                "1.8"))
+
+(def var-meta-whitelist
```

Comment:
Let's avoid the `whitelist` termnology where. I guess we can call this `safe-var-metadata` or something along those lines.

Reply:
(and I guess the original oversight was mine)